skills/pixiu-filter-author/scripts/gen-registry-import.sh

```
@@ -0,0 +1,66 @@
+  echo "error: $REG not found; set PIXIU_ROOT=<pixiu-repo-path>" >&2
+  exit 1
+fi
+
+MOD="github.com/apache/dubbo-go-pixiu"
+
+# Collect packages that register a filter.
+declare -a WANT=()
+while IFS= read -r gofile; do
+  if grep -q -E 'filter\.RegisterHttpFilter|filter\.RegisterNetworkFilterPlugin' "$gofile" 2>/dev/null; then
+    dir=$(dirname "$gofile")
+    rel="${dir#$ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)